woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -149,6 +149,17 @@ echo -n "Making the filesystem usable..." >/dev/console
 # busybox mount /proc before remounting / as rw (full install)
 busybox mount -t proc none /proc 2>/dev/null; STATUS=$((STATUS+$?))
 
+mkdir /sys 2>/dev/null
+busybox mount -t sysfs none /sys ;STATUS=$((STATUS+$?))
+
+#Update microcode if possible
+if [ "$(uname -m | grep -E '^x86_64|^amd64|^i.86')" != "" ] && [ "$(which lscpu)" != "" ]; then
+ #Make sure that cpu is the physical cpu or microcode was not loaded early by boot parameter
+ if [ "$(lscpu | grep "Hypervisor")" == "" ] && [ "$(dmesg | grep "microcode updated early")" == "" ]; then
```

Comment:
`dmesg` output can be long, better limit to one match with `-m 1 `, to stop that `grep` as soon as possible